src/functions.c

```diff
@@ -370,6 +370,61 @@ void fcallCommandReadOnly(client *c) {
     fcallCommandGeneric(c, 1);
 }
 
+void functionsDumpCommand(client *c) {
+    unsigned char buf[2];
+    uint64_t crc;
+    rio payload;
+    rioInitWithBuffer(&payload,sdsempty());
+
+    functionsSaveRio(&payload);
+
+    /* RDB version */
+    buf[0] = RDB_VERSION & 0xff;
+    buf[1] = (RDB_VERSION >> 8) & 0xff;
+    payload.io.buffer.ptr = sdscatlen(payload.io.buffer.ptr,buf,2);
+
+    /* CRC64 */
+    crc = crc64(0,(unsigned char*)payload.io.buffer.ptr,
+                sdslen(payload.io.buffer.ptr));
+    memrev64ifbe(&crc);
+    payload.io.buffer.ptr = sdscatlen(payload.io.buffer.ptr,&crc,8);
+
+    addReplyBulkSds(c,payload.io.buffer.ptr);
+}
+
+void functionsRestoreCommand(client *c) {
```

Comment:
```suggestion
void functionRestoreCommand(client *c) {
```

Reply:
Also missing the helpful documentation right above the function that indicates arguments :)

Reply:
Fixed, not resolving so you can review the documentation added.